src/python/pants/backend/codegen/protobuf/subsystem/protoc.py

```diff
@@ -10,8 +10,12 @@
 
 class Protoc(NativeTool):
   options_scope = 'protoc'
-  support_dir = 'bin/protobuf'
   default_version = '2.4.1'
 
   replaces_scope = 'gen.protoc'
   replaces_name = 'version'
+
+  # 'protoc' is also the binary's filename.
```

Comment:
Ditto, this should no longer be necessary, but might be worth deleting in a separate change.

Reply:
As above, I'll make a separate change after this gets merged doing that.